p8e-api/src/main/kotlin/io/provenance/engine/config/AppConfig.kt

```diff
@@ -261,4 +264,15 @@ class AppConfig : WebMvcConfigurer {
                 }.toMap()
             MetricsService(collectors, labels)
         }
+
+    /**
+     * Add support for new key management.
+     */
+    @Bean
+    fun signer(smartKeyProperties: SmartKeyProperties, signerProperties: SignerProperties) =
+       when(signerProperties.type.toLowerCase()) {
+            "default" -> Pen()
+            "smartkey" -> SmartKeySigner(smartKeyProperties.apiKey)
```

Comment:
Will improve the signer selection here with enum type, the SmartKeySigner() does not work with p8e just yet.